std/process.d

```diff
@@ -3233,11 +3242,15 @@ private:
     assertThrown(environment["std_process"]);
 
     // get() without default value
-    assert (environment.get("std_process") == null);
+    assert (environment.get("std_process") is null);
 
     // get() with default value
     assert (environment.get("std_process", "baz") == "baz");
 
+    // get() on an empty (but present) value
+    environment["std_process"] = "";
```

Comment:
yeah, it looks like wine has [some bugs](https://github.com/dlang/phobos/pull/4627/files#diff-8ac4ebb5dcc368ac0e9b7b9f118aa824R3259) with 0-length env variables.  i propose we restore the `version(Posix)` gate to this block.